src/app/footer/footer.component.html

```diff
@@ -59,7 +59,7 @@ <h4><a href="https://lindat.cz/partners">Partners</a></h4>
         <h4><a href="https://lindat.cz/services">Services</a></h4>
         <ul>
           <li><a href="https://lindat.mff.cuni.cz/en/monitoring">Service Status</a></li>
-          <li><a href="https://lindat.mff.cuni.cz/repository/xmlui/page/about?locale-attribute=en">About and Policies</a></li>
+          <li><a href="https://lindat.mff.cuni.cz/repository/static/about">About and Policies</a></li>
```

Comment:
can't this be relative?

Reply:
This `footer.html` is hard copied. Of course it could be dynamic, but it will take some time.